openvdb_houdini/houdini/SOP_OpenVDB_Combine.cc

```diff
@@ -1031,8 +1031,8 @@ struct SOP_OpenVDB_Combine::CombineOp
             // For level set grids, use the level set rebuild tool to both resample the
             // source grid to match the reference grid and to rebuild the resulting level set.
             const ValueT halfWidth = ((ref.getGridClass() == openvdb::GRID_LEVEL_SET)
-                ? ValueT(ZERO + this->getScalarBackgroundValue(ref) * (1.0 / ref.voxelSize()[0]))
-                : ValueT(src.background() * (1.0 / src.voxelSize()[0])));
+                ? ValueT(this->getScalarBackgroundValue(ref) * (1.0 / ref.voxelSize()[0]))
+                : src.background() * ValueT(1.0 / src.voxelSize()[0]));
```

Comment:
I've removed some of the zeroval + ... code from a few houdini based methods to resolve conversion warnings. As far as I can tell this was to support string grids? I can change this to a conditional if this needs to stay (the same way I've resolved pruneGrid in Grid.h)

Reply:
The `zeroVal<T>() + x` idiom allows values of arbitrary types to be supported, even types that can't be constructed with `T(x)`.  (For those types, one defines a `zeroVal` specialization and possibly an `operator+` overload.)  It's probably safe to remove it from Houdini code, which uses only a limited set of value types, but definitely not from, say, `Grid.h`.